src/ClusterBootstrap/params.py

```diff
@@ -397,7 +397,7 @@
     "DeployAuthentications": ["Corp", "Live", "Gmail"],
     # You should remove WinBindServers if you will use
     # UserGroups for authentication.
-    "WinbindServers": ["http://onenet40.redmond.corp.microsoft.com/domaininfo/GetUserId?userName={0}"],
+    # "WinbindServers": ["http://onenet40.redmond.corp.microsoft.com/domaininfo/GetUserId?userName={0}"],
```

Comment:
please check with anbang to make sure we don't need the default winbindservers.

Reply:
Confirmed with Anbang, this is internal. Are there any other internal default params that need deleting?